tsconfig.json

```diff
@@ -14,14 +14,14 @@
   ],
   "compilerOptions": {
     "target": "ES2020",
-    "module": "ESNext",
+    "module": "NodeNext",
```

Comment:
As we need to get rid of ESM in the native target, we need to handle both ESM and CJS output format. Which is supported by NodeNext.

Otherwise, @react-sping/native definition will not be found at code time and crashes on tests and lint.